web/blueprints/finance/__init__.py

```diff
@@ -235,6 +236,31 @@ def bank_accounts_import():
                            transactions=transactions,
                            old_transactions=old_transactions)
 
+@bp.route('/bank-accounts/importmanual', methods=['GET', 'POST'])
+@access.require('finance_change')
+def bank_accounts_import_manual():
+    form = BankAccountActivitiesImportManualForm()
+    form.account.choices = [(acc.id, acc.name) for acc in BankAccount.q.all()]
+
+    if form.validate_on_submit():
+        bank_account = BankAccount.q.get(form.account.data)
```

Comment:
Can be `bank_account = form.account.data` when using a QuerySelectField (see other comment).